open_schools_platform/user_management/users/services.py

```diff
@@ -106,6 +110,17 @@ def update_token_session(token: CreationToken, new_session: str) -> CreationToke
     return token
 
 
+def update_firebase_token_entity(*, token: FirebaseToken, data) -> FirebaseToken:
```

Comment:
what is data?
it is not clear

Reply:
Data may seem redundant here, because now the only field we can change in FirebaseToken model is token itself. But in the future timestamp field can be added for FirebaseToken, and we will need data here for convenient timestamp updating (just an example)

So I think data here is just more multipurpose.

And also data is a dict :)

Reply:
okay, it is ok